conf/config.yaml

```diff
@@ -65,6 +65,11 @@ nginx_config:                     # config for render the template to genarate n
     client_header_timeout: 60s     # timeout for reading client request header, then 408 (Request Time-out) error is returned to the client
     client_body_timeout: 60s       # timeout for reading client request body, then 408 (Request Time-out) error is returned to the client
     send_timeout: 10s              # timeout for transmitting a response to the client.then the connection is closed
+  server:                          # set custusmized directives in nginx.conf server field  
+    server_tokens: on              # Enables emitting nginx version on error pages and in the "Server"
```

Comment:
`server_tokens` is off in `http`, why set to `on` here?

Reply:
I just make a example here ，should I remove it？

Reply:
this file is NOT example file